react/react1/week3/src/components/ListItem.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import "./ListItem.css";
+import Border from "./Border";
+
+const ListItem = ({ description, id, handleDelete, deadline }) => {
+  const [done, setDone] = useState(false);
+  const [editing, setEditing] = useState(false);
+  const [taskDescription, setTaskDescription] = useState(description);
+  const handleDone = (id) => {
+    setDone(!done);
+  };
+
+  const handleSaveEdit = (e) => {
+    setEditing(!editing);
+    if (editing) {
+      setTaskDescription(taskDescription);
+    }
+    if (taskDescription === "") {
+      setTaskDescription(description); // Skip the update if taskDescription is empty 
+    }
```

Comment:
you can use it as ternary, otherwise you already have a description in default state value. I would suggest to check description for empty value by trimming it